listeners/increment.php

```diff
@@ -0,0 +1,9 @@
+<?php
+require "./services/api.php";
+
+function run($props, $event, $api)
+{
+    $counter = json_decode(getDoc($api, "counter", $props["id"])->getBody(), true);
```

Comment:
You should decode the JSON in the `getDoc` method:

```suggestion
    $counter = getDoc($api, "counter", $props["id"]);
```